tests/utils.py

```diff
@@ -194,11 +202,11 @@ def get_hf_assets(pattern, num_retry: int = 4, retry_delay: float = 30.0, check:
                         raise HTTPError(f"Impossible to parse XML file.") from e
             if not has_files:
                 raise HTTPError("No file downloaded.")
-        except HTTPError:
+        except HTTPError as e:
```

Comment:
This change is wrong. Remove it. It will raise the original exception after max number of trials anyway.